style.css

```diff
@@ -9,238 +8,560 @@
 }
 
 * {
-  margin: 0;
-
-  /* font-family: 'Playfair Display', serif; */
   font-family: 'Poppins', sans-serif;
+  width: auto;
+  margin: auto;
```

Comment:
- [ ] To prevent any auto layout preferences that may affect your work negatively, you need to set your `padding` and `margin` values to `zero`, and delete the `width: auto` property.